### PR TITLE
Pre-fill imgui.ini, change mappings for macos, add wasd to camera movement, allow actor layer selection, add build instructions and add undo/redo to menu bar.

### PR DESCRIPTION
**Pull request by Declan-Reid**
Makes sure that imgui.ini starts off with a pre-filled version so that it is less confusing upon intial opening.

### CODE REVIEW

**Comment by Declan-Reid** [comment]
Actors is not actually being selected

**Comment by Declan-Reid** [comment]
This now also adds the ability to choose a layer to add an actor to, and has build instructions in the readme (with a fancy lil warning).

**Comment by Declan-Reid** [comment]
This now also includes a way to change which layer an actor is on after being created. I also forgot to mention that undo and redo are in the menu bar in this.

File: Fushigi/Fushigi.csproj [hunk 1]
```diff
@@ -18,7 +18,9 @@
     <PackageReference Include="Silk.NET.OpenGL" Version="2.18.0" />
     <PackageReference Include="Silk.NET.Input" Version="2.18.0" />
     <PackageReference Include="Silk.NET.Input.Extensions" Version="2.18.0" />
+    <PackageReference Include="Silk.NET.OpenGL.Extensions.ImGui" Version="2.18.0" />
     <PackageReference Include="StbImageSharp" Version="2.27.13" />
+    <PackageReference Include="ZstdNet" Version="1.4.5" />
```
Comment (KillzXGaming) [inline Fushigi/Fushigi.csproj:23]: Is there a reason ZstdNet is included? I think the existing ZstdSharp package should be all that we need.

Comment (Declan-Reid) [reply Fushigi/Fushigi.csproj:23]: I don't remember putting this in to be honest.

**Comment by KillzXGaming** [comment]
Either ways the rest looks fine, so I will merge.

**Pull request merged**